testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -121,7 +121,7 @@ Feature: PXE boot a Retail terminal
     And I click on "Create Group"
     Then I should see a "System group HWTYPE:Intel-Genuine created." text
 
-  Scenario: Create terminal branch group
+  Scenario: Create branch terminals group
```

Comment:
So we create a terminal group here. Why do we use the word `branch` here?

Reply:
because imagine you have two branches:
- branch Seattle has terminals cash1, cash2, selfservice1, and selfservice2
- branch Miama has terminals cash1, cash2, cash3 and cash4

You would create two groups, one per branch, each with 4 terminals.

Reply:
Got it, thank you!